examples/binaries/stack-allocations/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "demo-stack-allocations"
+version = "0.1.0"
+authors.workspace = true
+edition = "2021"
+license = "GPL-3.0"
+workspace = "../../../"
+
+[dependencies]
+gstd.workspace = true
+codec = { package = "parity-scale-codec", version = "3.4.0", default-features = false, features = ["derive"] }
```

Comment:
workspace

Reply:
and below

Reply:
Cannot use workspace here - cannot understand why maybe some cargo problems

Reply:
OK, this one works.
But codec also should work, cannot understand what the problem with it 
https://github.com/gear-tech/gear/blob/f76565dfb63b3e79ee1949cf3dd9360b2b88af12/gtest/Cargo.toml#L18